packages/compose/src/hooks/use-focus-on-mount/index.js

```diff
@@ -79,6 +79,20 @@ export default function useFocusOnMount( focusOnMount = 'firstElement' ) {
 			return;
 		}
 
+		if ( typeof focusOnMountRef?.current === 'function' ) {
+			timerId.current = setTimeout( () => {
+				const tabbables = focus.tabbable.find( node );
+
+				const elementToFocus = focusOnMountRef.current( tabbables );
```

Comment:
What should happen if `tabbables` or `elementToFocus` is empty?

Reply:
Focus on close? I figure this works for when modal is a dialog for info only?

Reply:
I wonder if we should provide a better DX by `console.info` or something? Might be overthinking...